calculator.js

```diff
@@ -0,0 +1,36 @@
+var calculator = {
+	num1: 0,
+	num2: 0,
+	exp: 1,
```

Comment:
i dont think you need these 3 properties for your calculator object. What are they doing for you?